torch/nn/functional.py

```diff
@@ -523,7 +526,10 @@ def fractional_max_pool3d_with_indices(
     return_indices: bool = False,
     _random_samples: Optional[Tensor] = None
 ) -> Tuple[Tensor, Tensor]:
-    r"""Applies 3D fractional max pooling over an input signal composed of several input planes.
+    r"""
+    fractional_max_pool3d(input, kernel_size, output_size=None, output_ratio=None, return_indices=False)
```

Comment:
We actually have `_random_samples` in the signatures for the other functions (even though they're not documented). Not sure if we want it here for consistency as well or not, your call!